uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/leave-status-id-model.js

```diff
@@ -0,0 +1,39 @@
+define([
+  'leave-absences/shared/modules/models',
+  'common/services/api/option-group',
+  'common/models/model'
+], function (models) {
+  'use strict';
+
+  models.factory('LeaveStatusID', [
```

Comment:
Why is this necessary? You are creating a service that is only used internally by a `LeaveRequestInstance`. Then why simply not having the instance using internally (maybe on `init()`?) the `OptionGroup` service to fetch the option values related to the status?

I'd like to avoid having an over-populate library of services, we should keep them to a minimum